Makefile

```diff
@@ -1,4 +1,24 @@
+
+IMAGE=segment/mongodb-source
+VERSION=v0.1.3-beta
+
+save-deps:
+	godep save
+
+run:
+	go run main.go mongo.go collection.go config.go description.go $(ARGS)
+
 build:
```

Comment:
+1 to tejas's comment, also let's see if we can find a way to leave out hardcoded filenames in the Makefile. would rather have it out of here than have hard to maintain code